detox/src/devices/drivers/android/exec/ADB.js

```diff
@@ -271,13 +271,16 @@ class ADB {
   }
 
   // TODO refactor the whole thing so as to make usage of BinaryExec -- similar to EmulatorExec
-  async adbCmd(deviceId, params, options) {
+  async adbCmd(deviceId, params, options = {}) {
     const serial = `${deviceId ? `-s ${deviceId}` : ''}`;
     const cmd = `"${this.adbBin}" ${serial} ${params}`;
-    const retries = _.get(options, 'retries', 1);
-    _.unset(options, 'retries');
+    const _options = {
+      ...options,
+      retries: options.retries || 1,
```

Comment:
I'm afraid that `|| 1` is not an equivalent transformation. Just so you are aware that you no longer will be able to nullify retry count, i.e. `{ retries: 0} -> { retries: 1 }`.

Nevertheless, this function has never been called with zero, so we are in a safe position.

Reply:
What an excellent find. I will fix this + add a unit test!